backend/src/main/java/gov/cdc/usds/simplereport/service/AzureStorageQueueTestEventReportingService.java

```diff
@@ -79,7 +79,9 @@ public void markTestEventsAsReported(Set<TestEvent> testEvents) {
 
   private String toBuffer(TestEvent testEvent) {
     try {
-      return mapper.writeValueAsString(new TestEventExport(testEvent));
+      return mapper
+          .writeValueAsString(new TestEventExport(testEvent))
+          .replaceAll("[\u2028\u2029]", "");
```

Comment:
oh I just saw you called this out
>Are there any other characters, whitespace or otherwise, we would like to account for in this manner?

I'm trying to remember what method failed because of these characters. I was thinking we could use that as a jumping-off point to determine if there's anything else that would cause the same parse failure
@DanielSass do you remember?

Reply:
What about this one https://www.compart.com/en/unicode/U+2063?

Reply:
> I'm trying to remember what method failed because of these characters. I was thinking we could use that as a jumping-off point to determine if there's anything else that would cause the same parse failure

Sorry I didn't see the notification for this until today 🤐.  There wasn't a method that failed within our code - it was just the API talking to Report Stream.